src/monodroid/CMakeLists.txt

```diff
@@ -196,10 +197,12 @@ else()
       endif()
     endif()
 
-    link_directories("${XA_LIB_TOP_DIR}/${ANDROID_ABI}")
+	set(XA_LIBRARY_OUTPUT_DIRECTORY "${XA_LIB_TOP_DIR}/${ANDROID_ABI}")
+	link_directories("${XA_LIBRARY_OUTPUT_DIRECTORY}")
   endif()
 
   if(DEFINED HOST_BUILD_NAME)
+	set(XA_LIBRARY_OUTPUT_DIRECTORY "${XA_LIB_TOP_DIR}/${HOST_BUILD_NAME}")
```

Comment:
Please correct indentation.  Looks like you have Emacs set to turn 4 spaces into tabs, instead of just always expanding tabs in this file.